core/sql/common/multi-byte.h

```diff
@@ -18,12 +18,6 @@
 // under the License.
 //
 // @@@ END COPYRIGHT @@@
-/*
```

Comment:
Hewlett Packard donated the original Trafodion code to Apache.  HP is a founding member of the Open Software Foundation and has all rights to code produced under this license.  Therefore these copyrights are covered by the same agreement that allows us to remove HP copyrights.  This information came from a conversion I had with the team that handled the software donation from HP to Apache.